src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java

```diff
@@ -56,6 +56,8 @@
 
 public class AWSCredentialsImpl extends BaseAmazonWebServicesCredentials implements AmazonWebServicesCredentials {
 
+    private static final long serialVersionUID = -8073785642456657271L;
```

Comment:
I generated it using the generator in intellij from the 1.16 release tag. How'd you generate yours?